blocks/library/post-title/post-title-edit.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import Editable from '../../editable';
+import PostPermaLink from './post-permalink';
+import { editPost } from '../../../editor/actions';
+
+const PostTitleEdit = ( { title, focus, setFocus, onChange } ) => [
+	focus && <PostPermaLink />,
+	<Editable
+		key="editable"
+		tagName="h1"
+		value={ [ title ] }
+		focus={ focus }
+		onFocus={ setFocus }
+		onChange={ onChange }
+		style={ { textAlign: 'left' } }
+		formattingControls={ [] } /> ];
+
+export default connect( null, dispatch => ( { onChange: title => {
```

Comment:
This line is a bit difficult to understand given the many levels of function nesting and differing implicit return / explicit arrow function blocks it creates.